ethcore/src/spec/spec.rs

```diff
@@ -123,6 +156,8 @@ pub struct CommonParams {
 	pub max_code_size_transition: BlockNumber,
 	/// Transaction permission managing contract address.
 	pub transaction_permission_contract: Option<Address>,
+	/// Irregular state change list.
```

Comment:
Would be good to mention that the irregular state changes are applied at the very beginning of the block to avoid confusion.